lib/python/picongpu/picmi/requirements.txt

```diff
@@ -0,0 +1,6 @@
+typeguard >= 4.2.1
```

Comment:
you must update https://github.com/ComputationalRadiationPhysics/picongpu/blob/800f7bc119d09a3d629aa34a80e2ff368d4289e5/share/ci/install/pypicongpu.sh#L43

to point to this file because the dependency generator can not handle the link in file `lib/python/picongpu/requirements.txt`

Reply:
Thnak you @psychocoderHPC